### DISCUSSION TITLE
ppignore

### DISCUSSION THREAD

**Discussion by Ball-Pic** [photoprism/photoprism#3476] [General]
Hello
I would like to exclude certain sub-folders from indexing in my photo directory.
I would like to do that to add these folders with [#NoIndex] at the end of the file directoryname.
Such as "Photo Trip [#NoIndex]".
How do I have to put the search term in ppignore?

**Comment by lastzero** [comment]
You should not use # in file and folder names:

https://www.mtu.edu/umc/services/websites/writing/characters-avoid/

**Reply by Ball-Pic** [reply]
Thank you for your fast responce....
I changed the names without '#' like "Photo Trip [NoIndex]"
In ppignore ist this syntax correct?:
![image](https://github.com/photoprism/photoprism/assets/81575941/5cec5429-a04c-4da5-9598-449980457c98)

**Reply by lastzero** [reply]
You should test it first because sometimes [ is also treated as a special character.

**Reply by Ball-Pic** [reply]
Sorry - I've already noticed.
without [ it works as described.

However, I have one more question: if I set up the definition in ppignore in the root folder, should it also apply to all sub-folders, or do I have to set the file for all sub-folders? Or is there a parameter that can be set so that the setting in ppignore is passed on to all subfolders?

**Reply by lastzero** [reply]
Yes, patterns also apply to subfolders unless, for example, you prefix filenames with a path.

**Reply by Ball-Pic** [reply]
Yes, patterns also apply to subfolders unless, for example, you prefix filenames with a path.

**Reply by Ball-Pic** [reply]
Files in the subdirectories are no longer read in. Can it be that the subdirectories are created?
These were created during my test, but are empty
![image](https://github.com/photoprism/photoprism/assets/81575941/326d4de7-84c0-4fd2-a7da-625c6ab5d026)
-->
![image](https://github.com/photoprism/photoprism/assets/81575941/04ee9a13-0778-424f-98e4-92cd6edd8c3f)
-->
![image](https://github.com/photoprism/photoprism/assets/81575941/9f780152-4b42-42d9-94ff-5e9373f3a39b)
![image](https://github.com/photoprism/photoprism/assets/81575941/7a351c78-8077-48dc-ab25-4a5a7fd79006)
